src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputTryCatchIfElse2.java

```diff
@@ -0,0 +1,47 @@
+package com.google.checkstyle.test.chapter4formatting.rule412nonemptyblocks;
+
+/** some javadoc. */
+public class InputTryCatchIfElse2 {
+  /** some javadoc. */
+  public static void main(String[] args) {
+    boolean after = false;
+    try {
+      /* foo */
+    } finally { after = true; }
+    // 2 violations above:
+    //  ''{' at column 15 should have line break after.'
+    //  ''}' at column 31 should be alone on a line.'
+
+    try {
+      /* foo */
+    } catch (NullPointerException e) {
+      // violation below ''}' at column 5 should be alone on a line.'
+    } catch (Exception e) {
+      // violation below ''}' at column 5 should be alone on a line.'
+    } finally {
+      after = true;
+    }
```

Comment:
`// violation below ''}' at column 5 should be alone on a line.'` violations are false positive, this is happening with this try/catch only. Following try/catch seems to be working fine:

https://github.com/checkstyle/checkstyle/blob/c0cbda5c074359393ad2d5e2f05a3bfbfa2b6358/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputTryCatchIfElse.java#L37-L45